service/src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -286,6 +284,22 @@ public void deleteWorkspace(Workspace workspace, AuthenticatedUserRequest userRe
     deleteJob.submitAndWait(null);
   }
 
+  /**
+   * Update workspace properties.
+   *
+   * @param userRequest authenticated user
+   * @param workspaceUuid workspace of interest
+   * @param properties map of key-value properties
+   */
+  public void updateWorkspaceProperties(
+      AuthenticatedUserRequest userRequest, UUID workspaceUuid, Map<String, String> properties) {
+    validateWorkspaceAndAction(userRequest, workspaceUuid, SamConstants.SamWorkspaceAction.WRITE);
```

Comment:
As of Zach's PR, we are moving all access checks into the controller. This should be moved there.